runner/local/simple.go

```diff
@@ -137,12 +138,16 @@ func (r *simpleRunner) run(cmd *runner.Command, runId runner.RunId, doneCh chan
 	// Wait for checkout or cancel
 	select {
 	case <-doneCh:
+		go func() {
```

Comment:
Now that I look at it, it seems like we would get a nil pointer exception if doneCh closes and 'defer checkout.Release()' gets called before checkout completes. The simplest solution would be to 'defer func() { if checkout != nil { checkout.Release() } }()